### ISSUE TITLE
Stopped working after update

### ISSUE THREAD

**Issue by drumanshoo** [Synthoid/ExportSheetData#12]
Stopped working after upgrade. Just spins and doesn't bring up JSON. I am doing it for the current sheet and all other fields are the default.

Console error: Uncaught Error: We're sorry, a server error occurred. Please wait a bit and try again.(…)

Have been getting this since yesterday.

**Comment by Synthoid** [comment]
Just did a quick smoke test on a few sheets. I'm not seeing that error. Can you answer a few questions to help me solve it?

1. Are you the sheet owner?
2. If not, is the folder containing the sheet shared with you, or just the sheet itself?
3. If you copy the sheet and try to export the copy, does it work?
4. If the answer is not immediately forthcoming, would you be able to send a copy of the sheet or give a sheet with a similar format for testing?

The server error issue is odd, normally the error message is more specific as to what the error is (ie "There is no function getName() for unidentified object").

In any case, please try answering those questions and let me know what happens. I should be able to do more afterwards.

**Comment by Synthoid** [comment]
I've rolled things back to the previous version for now, so this issue will be closed. I may need to revise my testing methods, since it seems like quite a few people had the same issue, but clearly I missed it.

**Comment by drumanshoo** [comment]
Synthoid, thanks for the response. I didn't notice it yesterday.

I later discovered that the file was being created in Drive, just not being
displayed.

Do I need to uninstall and then reinstall?

Thanks for your attentiveness!

To answer your questions, I did own the file and I did try copying the data
into a new sheet and experienced the same thing. I was able to "visualize".

Again, thanks!

Abundant returns,

Dru

Dru Shockley
*Call To Action Alliance*
303-783-5760
ctaalliance.com <http://www.ctaalliance.com>

marketing strategy to reward business passion | websites to ignite action
*passion ignites action*
